### DISCUSSION TITLE
Vtk vs Paraview

### DISCUSSION THREAD

**Discussion by antillgrp** [Kitware/trame#306] [Q&A] [answered]
Please illustrate (even better if there is something I can read) me what would be the difference between creating an app for remote rendering  based on vtk, thank one based on paraview?

Thanks

**Comment by jourdain** [comment]
- [VTK example](https://github.com/Kitware/trame/blob/master/examples/06_vtk/01_SimpleCone/RemoteRendering.py) (please ignore the event stuff, those are optional to show how it can be done)
- [ParaView example](https://github.com/Kitware/trame/blob/master/examples/07_paraview/SimpleCone/RemoteRendering.py)

**Reply by antillgrp** [reply]
Thanks, but I'm trying to understand how vtk and paraview differ, what should I take into consideration at the time in going in one direction or the other one. For example, in terms of performance and local-remote transfer.

**Reply by jourdain** [reply, marked as answer]
The difference is only the way you write the Python code since ParaView use proxy and is capable of being distributed (MPI). Also the ParaView trace can help you convert UI interaction into Python code that can be used as-is in trame.

For the trame/network side, there is no difference.
